content/en/account_management/billing/usage_attribution.md

```diff
@@ -87,7 +83,6 @@ This section provides daily reports at an hourly granularity to dig into time fr
 
 Daily data can also be pulled using the tool's public API. For more information, see the [API endpoint documentation][2].
 
```

Comment:
```suggestion
[2]: https://docs.datadoghq.com/api/v1/usage-metering/#get-hourly-usage-attribution
```
Moving this link back. Unfortunately, our pre-commit hook currently breaks these links by moving them down in the document. I have a PR up to fix it.